### ISSUE TITLE
Backplane Mechanical Issues

### ISSUE THREAD

**Issue by gravelpitman** [sinara-hw/DIOT_Backplane#2]
### **Incorrect mounting hole spacing** 

![image](https://github.com/user-attachments/assets/f60ed63a-3a39-4f6b-a13d-9cd957890305) 

Crate dictates a hole spacing of 122,1 as shown above 

![image](https://github.com/user-attachments/assets/6b287cf4-0c17-44d8-a8b2-1bad8a02face) 

Hole spacing in PCB shown above.  

The difference of 0.4mm is too big in my opinion.  

### **Placement of the coordinate system** 

![BackplaneOrgin](https://github.com/user-attachments/assets/d6a71a68-81fd-4d9f-8cde-93b16d1cb52e) 

This placement does not make sense. In my opinion, the location of the coordinate system should be in the mounting hole, for example, in the lower left mounting hole.  This location provides any possibility to calculate the correct location of connectors, (16,6mm between modules and backplane orgin i Y axis)

  

### **Not using meaningful units of measurement (mm)** 

![image](https://github.com/user-attachments/assets/29e78ab7-e192-4589-a110-8ef3a5ea416e) 

Please use max 0,01mm incremenst (thousandths of a millimeter) 

### **-Correct holes placement**
![image](https://github.com/user-attachments/assets/a19786e9-c621-4a95-a073-b686e0b2f14f)
Please make them as shown above. 

### **-TOP connectors**
![image](https://github.com/user-attachments/assets/3e659a9b-eb11-465c-8a1f-7605ef86e811)
Every dimension measured between the connector orgin and PCB orgin
### **-TOP Guides**
![image](https://github.com/user-attachments/assets/2d6e066d-e1a7-4090-b72d-893f8d471d94)
Guide X axis orgin same as mounting holes
Every dimension measured between the guide orgin and PCB orgin
### **-BOTTOM connectors**
![image](https://github.com/user-attachments/assets/5bccc1e2-e4dc-4a97-9344-9621ed9631fe)
Every dimension measured between the connector orgin and PCB orgin
### **-BOTTOM Guides**
![image](https://github.com/user-attachments/assets/9e883314-9251-4cdc-9b8f-3238756b75f5)
Every dimension measured between the guide orgin and PCB orgin
### **The assumptions I made regarding crate assembley**

1. Modules inserted symmetrically
![image](https://github.com/user-attachments/assets/817c924f-5298-47cc-9a09-de9324d525d2)
The picture shows symmetrical clearance in the X axis of the entire crate and symmetrical clearance i Y axis of crate

2. Maximum Misalignment of AMPHENOL 51940-650 according to the manufacturer's designation 
(datasheet: https://cdn.amphenol-cs.com/media/wysiwyg/files/documentation/bus-20-067.pdf , 13/24)

**Comment by filipswit** [comment]
@gravelpitman can you verify mechanics? 

[DIOT_Backplane_v1([No Variations]).zip](https://github.com/user-attachments/files/17481800/DIOT_Backplane_v1.No.Variations.zip)

**Comment by gravelpitman** [comment]
@filipswit 
Now placement of connectors is correct.
My only complain is the positioning of the top and bottom edges in relation to the PCB orgin.
![image](https://github.com/user-attachments/assets/fe84d890-823e-4b3f-9a74-4508ebd996a3)
Please move only edges of PCB.

**Comment by filipswit** [comment]
Corrected edges placement.

**Comment by jordens** [comment]
Also, the 7 Würth terminals (not mounted) are not on a grid. Could they be snapped to a 30 mm or to a 6 HP grid?

**Comment by gravelpitman** [comment]
Hi, 
Sorry for not replying for so long. I’ve made a special tool checking the disputed bolt spacing. I checked hole spacing in DIOT crate, Sinara (NO fun unit) etc. I admit that I was wrong. Bolt spacing is in accordance with the standard 122,5mm. 
However, I stand by the remarks regarding the location of the PCB origin etc.
I am curious where the discrepancy between the model and reality comes from. Will try to get an answer from Schroff.

**Comment by kaolpr** [comment]
@gravelpitman Can we consider it closed?

**Comment by gravelpitman** [comment]
No. As mentioned in my previous comment, I made an error – the Y-axis bolt spacing should be 122.5 mm, as specified in Jordan’s comment from November 11, 2024. All connectors are correctly positioned; only the mounting holes need to be relocated.

Sorry for that mistake. I really don't know why the 3D models of Schroff crates are incorrect.

**Comment by jordens** [comment]
The mistake is in the CAD models. Sidewalls have 121.55 bolt hole separation. Horizontal rails have 0.275 offset between sidewall and frontpanel bolt holes. That's 121.55 + 2*0.275 = 122.1 panel bolt hole centerline separation. In the assembled CAD models for crate kits from Schroff, the rails are intentionally misaligned by 0.2 w.r.t. the sidewalls bolt holes to fix that.

![Image](https://github.com/user-attachments/assets/1c801264-57f4-411d-81d7-3ddd4bd30d25)

**Comment by martafafara** [comment]
I moved mounting holes into (hopefully) proper position. @gravelpitman received step file to final verification. If passed, please close the issue.

**Comment by gravelpitman** [comment]
I’ve reviewed the backplane’s mechanical design, and everything appears to be correct. I’m now closing this issue.

**Comment by jordens** [comment]
I took the liberty of posting the CAD assembly issue over at https://ohwr.org/project/diot-crate-hw/-/issues/119
